files/requirements.py

```diff
@@ -80,6 +80,12 @@ def setup_python(python: Python, source_directory: str, final: bool) -> None:
 
     for requirements in requirements_list:
         display.section(f'Installing requirements for Python {python.version} ({requirements})')
+
+        # Create file for new Python versions
+        if not os.path.exists(os.path.join(source_directory, requirements)):
+            with open(os.open(os.path.join(source_directory, requirements), os.O_CREAT)):
+                pass
+
```

Comment:
This shouldn't be needed, as long as you run `make freeze` before building the container.